README.md

```diff
@@ -77,6 +73,29 @@ alias for the latest release from an LTS line. For example, the
 [latest-carbon](https://nodejs.org/download/release/latest-carbon/) directory
 contains the latest Carbon (Node.js 8) release.
 
+
+#### Alternative Ways to Install:
```

Comment:
If we're going to add these, we should probably first break them down by platform. Even then we should only include *officially supported* sources/methods.

Reply:
And they should probably be in a separate PR.